tests/monitor/monitor.test.mjs

```diff
@@ -29,9 +31,8 @@ describe('setup', () => {
       projectRootPath,
       'tests/monitor/docker-compose.test.yml',
     )
-      .withBuild()
```

Comment:
we don't need to build everytime like we do in ar-io-node, this ends up costing 2-5 mins on every execution